src/pages/Home/index.tsx

```diff
@@ -66,6 +68,31 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
 
   // console.log(order, 'this is order3'); // this is for you, u said you want to see the console.log
 
+  useEffect(() => {
+    fetch('http://127.0.0.1:8000/boba_list')
+    .then(response => {
+      if (response.ok){
+        return response.json()
+      }
+      throw response;
```

Comment:
You want to handle response not 200 with redirect or display error. Not crashing the site if server is not available.

Reply:
updated at f738575ddca78aaaf7ab71c70f70a0047e756432